waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -236,6 +237,30 @@ genRoutesDir spec =
         )
     ]
 
+genTypesAndEntitiesDirs :: AppSpec -> Generator [FileDraft]
+genTypesAndEntitiesDirs spec = return [entitiesIndexFileDraft, typesIndexFileDraft]
+  where
+    entitiesIndexFileDraft =
+      C.mkTmplFdWithDstAndData
+        [relfile|src/entities/index.ts|]
+        [relfile|src/entities/index.ts|]
+        (Just $ object ["entities" .= allEntities])
+    typesIndexFileDraft =
+      C.mkTmplFdWithDstAndData
+        [relfile|src/types/index.ts|]
+        [relfile|src/types/index.ts|]
+        ( Just $
+            object
+              [ "entities" .= allEntities,
+                "isAuthEnabled" .= isJust userEntityName,
+                "userEntityName" .= fromMaybe "" userEntityName,
+                "userViewName" .= fromMaybe "" userViewName
+              ]
+        )
+    allEntities = map (C.buildEntityData . fst) $ AS.getDecls @AS.Entity.Entity spec
+    userEntityName = AS.refName . AS.App.Auth.userEntity <$> AS.App.auth (snd $ getApp spec)
+    userViewName = (++ "View") <$> userEntityName
```

Comment:
This one feels like a candidate for extracting at some point and going more global. But ok for now!

Reply:
Yes, same thing with `"password"`. I'll add a comment.